src/compiler/crystal/codegen/fun.cr

```diff
@@ -98,7 +98,7 @@ class Crystal::CodeGenVisitor
         {% if LibLLVM::IS_LT_150 %}
           context.fun.add_attribute LLVM::Attribute::UWTable
         {% else %}
-          context.fun.add_attribute LLVM::Attribute::UWTable, value: LLVM::UWTableKind::Async
+          context.fun.add_attribute LLVM::Attribute::UWTable, value: LLVM::UWTableKind::Sync
```

Comment:
Would it make sense to make it async except for macOS?

Reply:
It's actually ARM64 Linux too, I'll try